### PR TITLE
[docs] Add backlinks from hyperopt / rl algorithm examples to the built-on Ray libraries

### PR DESCRIPTION
**Pull request by ericl**
## What do these changes do?

Seems like a lot of visitors land on those pages, but now that we have libraries we should link them there.

### CODE REVIEW

File: doc/source/example-a3c.rst [hunk 1]
```diff
@@ -11,6 +11,10 @@ View the `code for this example`_.
 .. _`Universe Starter Agent`: https://github.com/openai/universe-starter-agent
 .. _`code for this example`: https://github.com/ray-project/ray/tree/master/python/ray/rllib/a3c
 
+.. note::
+
+    For a more comprehensive framework of reinforcement learning in Ray, see `Ray RLlib <http://ray.readthedocs.io/en/latest/rllib.html>`__.
```
Comment (ericl) [inline doc/source/example-a3c.rst]: (Wording suggestions welcome)

Comment (robertnishihara) [reply doc/source/example-a3c.rst]: How about, "For an overview of Ray's reinforcement learning library, see ..."

File: doc/source/example-policy-gradient.rst [hunk 2]
```diff
@@ -4,6 +4,11 @@ Policy Gradient Methods
 This code shows how to do reinforcement learning with policy gradient methods.
 View the `code for this example`_.
 
+.. note::
+
+    For a more comprehensive framework of reinforcement learning in Ray, see `Ray RLlib <http://ray.readthedocs.io/en/latest/rllib.html>`__.
```
Comment (robertnishihara) [inline doc/source/example-policy-gradient.rst]: same as above

File: doc/source/example-a3c.rst (hunk 1 above)
Comment (ericl) [reply doc/source/example-a3c.rst]: I like it. Updated.

File: doc/source/example-policy-gradient.rst (hunk 2 above)
Comment (ericl) [reply doc/source/example-policy-gradient.rst]: Updated.

**Review by richardliaw — APPROVED** [review]
lgtm

**Pull request merged**